src/label/Label.ts

```diff
@@ -45,12 +44,6 @@ export const LabelBase = ThemeableMixin(WidgetBase);
 
 @theme(baseCss)
 export default class Label extends LabelBase<LabelProperties>  {
-	@onPropertiesChanged()
-	protected onPropertiesChanged(event: PropertiesChangeEvent<this, LabelProperties>) {
```

Comment:
Do we no longer need to pass a registry down to Label? I think @smhigley had some issues when wrapping a Select that uses a custom registry.